### PR TITLE
Update notices.json

### PR DESCRIPTION
**Pull request by jgilliam17**
notices test

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [
 {
-    "id": "9",
+    "id": "10",
     "conditions": {
       "audience": "all",
```
Comment (jgilliam17) [inline notices.json:5]:
```suggestion
      "audience": "member",
```

File: notices.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 [
 {
-    "id": "9",
+    "id": "10",
     "conditions": {
-      "audience": "all",
-      "clientType": "desktop"
+      "audience": "member",
+      "clientType": "web"
```
Comment (jgilliam17) [inline notices.json:6]:
```suggestion
```

File: notices.json [hunk 3]
```diff
@@ -1,9 +1,8 @@
 [
 {
-    "id": "9",
+    "id": "10",
     "conditions": {
-      "audience": "all",
-      "clientType": "desktop"
+      "audience": "member",
```
Comment (jgilliam17) [inline notices.json:5]:
```suggestion
      "audience": "all",
```

File: notices.json [hunk 4]
```diff
@@ -15,14 +14,15 @@
     }
   },
   {
-    "id": "8",
+    "id": "11",
+      "repeatable": "true",
```
Comment (jgilliam17) [inline notices.json:18]:
```suggestion

```

**Comment by jaydeland** [comment]
@jgilliam17 - here is the error:
$ ajv validate -s notices.schema.json -d notices.json
error:  /builds/mirrors/core/notices/notices.json: Unexpected token } in JSON at position 69
ERROR: Job failed: command terminated with exit code 2

**Pull request closed without merging**